BuildResidentialHPXML/measure.rb

```diff
@@ -1076,6 +1073,18 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDescription("Type of air leakage. If '#{HPXML::InfiltrationTypeUnitTotal}', represents the total infiltration to the unit as measured by a compartmentalization test, in which case the air leakage value will be adjusted by the ratio of exterior envelope surface area to total envelope surface area. Otherwise, if '#{HPXML::InfiltrationTypeUnitExterior}', represents the infiltration to the unit from outside only as measured by a guarded test. Required when unit type is #{HPXML::ResidentialTypeSFA} or #{HPXML::ResidentialTypeApartment}.")
     args << arg
 
+    leakiness_description_choices = OpenStudio::StringVector.new
+    leakiness_description_choices << HPXML::LeakinessVeryTight
+    leakiness_description_choices << HPXML::LeakinessTight
+    leakiness_description_choices << HPXML::LeakinessAverage
+    leakiness_description_choices << HPXML::LeakinessLeaky
+    leakiness_description_choices << HPXML::LeakinessVeryLeaky
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('leakiness_description', leakiness_description_choices, false)
+    arg.setDisplayName('Air Leakage: Leakiness Description')
+    arg.setDescription('Qualitative description of infiltration. If provided, Manual J Table 5A/5B look up ACH values will be used for hvac sizing, and the annual average ACH will be calculated based on year built, iecc zone, cfa, infiltration height, foundation type, ducts and leakiness description.')
+    args << arg
```

Comment:
A few suggestions:
1. Move this input above the other air leakage inputs, since it's the more generic input.
2. Use a name consistent with the other other air leakage arguments: `air_leakage_leakiness_description`
3. No need for a detailed description of what will happen for Manual J and the energy model here, that should be in the documentation.
4. In the code below, let's use the quantitative input if it's provided, else the qualitative input. I don't think we need to support the unusual use case where someone wants to provide both. And we should update the description of the quantitative input to say that it will override the qualitative input if provided.
5. Since the quantitative defaults were removed, there are now no defaults at all for infiltration. A user of the measure will therefore hit an error unless they specifically enter something. In general, we'd like to have reasonable defaults. I suggest we put back in a default -- probably the qualitative input of "average" is a good default.

Reply:
I have a question, if quantitative input is overriding the qualitative input, and with default assumptions in the place, then it will always have values in quantitative arguments no matter if it's defaulted or user-provided, how can we get the qualitative input?

Reply:
> I have a question, if quantitative input is overriding the qualitative input, and with default assumptions in the place, then it will always have values in quantitative arguments no matter if it's defaulted or user-provided, how can we get the qualitative input?

Sorry, I don't think I'm understanding. You removed the default quantitative inputs, and I was suggested "average" as the qualitative input. So if they leave everything as is, or if they change the qualitative input (to e.g. "tight"), then the HPXML will only have that value.

Reply:
I see, I thought you mean putting back all the default values (both quantitative and qualitative), then we're in agreement to only allow 'average' as default.